rest/rest_endpoints.go

```diff
@@ -60,6 +60,7 @@ var (
 	CreateGuildChannel     = NewEndpoint(http.MethodPost, "/guilds/{guild.id}/channels")
 	GetGuildChannels       = NewEndpoint(http.MethodGet, "/guilds/{guild.id}/channels")
 	UpdateChannelPositions = NewEndpoint(http.MethodPatch, "/guilds/{guild.id}/channels")
+	GetActiveGuildThreads  = NewEndpoint(http.MethodGet, "/guilds/{guild.id}/threads/active")
```

Comment:
can you please move this into the part with Thread endpoints? L181

Reply:
Sure! Just to confirm, this is the case even though it is part of the Guilds endpoints on Discord's end?

Reply:
correct - we separate them like this even though they share a common endpoint